docs/book/src/reference/using_an_external_type.md

```diff
@@ -0,0 +1,155 @@
+# Using an External Type
+
+There are several different external types that may be referenced when writing a controller.
+* Custom Resource Definitions (CRD) that is defined in the current project `kubebuilder create api`.
+* Core Kubernetes Resources eg. Deployments or Pods.
+* CRD's that are created and installed in another project.
+* A Custom Resource defined via an API Aggregation (AA). Aggregated APIs are subordinate APIServers that sit behind the primary API server, which acts as a proxy.
+
+Currently Kubebuilder handles the first two, CRDs and Core Resources, seamlessly.  External CRDs and CRs created via Aggregation must be scaffolded manually.
+
+In order to use a Kubernetes Custom Resource that has been defined in another project
+you will need to have several items of information.
+* The Domain of the CR
+* The Group under the Domain 
+* The Go import path of the CR Type definition.
+
+The Domain and Group variables have been discussed in other parts of the documentation.  The import path would be located in the project that installs the CR.
+
+This document uses `my` and `their` prefixes as a naming convention for repos, groups, and types to clearly distinguish between your own project and the external one you are referencing.
```

Comment:
By default, we no longer have the group dir. Unless the project is configured to use multigroup. (ihmo) we need to explain it. Also, might not be necessary in all cases to have multi-group support to be able to reconcile an eternal type and the external type either might not have the group path

Reply:
Agreed. Rewrote the guide to not depend on multi-group layouts since it is not necessary for writing this guide. I think the guidance can be written with one single controller and then people can combine the guides together if they wish to have them coexist.